plugin/al/mayautils/AL/maya/tests/mayaplugintest/utils/NodeHelperUnitTest.cpp

```diff
@@ -96,6 +96,7 @@ MStatus NodeHelperUnitTest::initialise()
   {
     setNodeType(kTypeName);
 
+    addFrame("Fabrice");
```

Comment:
No, this actually fixes a crash.  If you look at the next few chunks of code you'll see that all of them add a frame before creating UI that is parented to them.  Not starting out with a Frame here is a crash on the next line when we create the bool attribute.  The other chunks of code use frame names like "hello" and "world", they don't mean anything at all, and when I was looking at the line of code I noticed that the name of the developer who added the line that was crashing was "Fabrice" so that's I went with it :-)